beetmoverscript/script.py

```diff
@@ -82,6 +89,27 @@
     await retry_download(context=context, url=source, path=abs_file_path)
     await retry_upload(context=context, destinations=destinations, path=abs_file_path)
 
+    if context.task['payload']['update_manifest']:
+        enrich_balrog_manifest(context, abs_file_path, locale)
```

Comment:
could be wrong but wouldn't this add every artifact (beet) to the balrog manifest? I'm pretty sure we don't need all the artifacts like checksums and mochitest.tests.zip for example